app/javascript/packs/MainApp/components/Referees.jsx

```diff
@@ -20,7 +20,17 @@ class Referees extends Component {
   }
 
   componentDidMount() {
-    this.handleSearch()
+    axios
+      .get('api/v1/national_governing_bodies')
+      .then(({ data: { data } }) => {
+        this.setState({
```

Comment:
Mind defining and calling a method here? That way the method name can explain what where doing instead of having the code under the generic “componentDidMount“ name.